_stbt/core.py

```diff
@@ -2614,6 +2615,9 @@ def _find_user_file(filename):
     :returns: Absolute filename, or None if it can't find the file.
     """
 
+    if isinstance(filename, unicode):
+        filename = filename.encode(sys.getfilesystemencoding(), "replace")
```

Comment:
I feel like there's less to go wrong if we just hard-code utf-8 here.  It's Linux only afterall.

Reply:
Fixed.